### PR TITLE
Documentation of coreg_fa

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Documentation of the function coreg_fa. Description and inputs in need of completion.

### CODE REVIEW

File: ea_coreg_fa.m [hunk 1]
```diff
@@ -1,4 +1,18 @@
 function ea_coreg_fa(options,usebrainmask)
+%
```
Comment (Daniel-Tojal) [inline ea_coreg_fa.m]: Please provide a description of the function.

File: ea_coreg_fa.m [hunk 2]
```diff
@@ -1,4 +1,18 @@
 function ea_coreg_fa(options,usebrainmask)
+% 
+%
+% USAGE:
+%
+%    ea_coreg_fa(options,usebrainmask)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_coreg_fa.m]: Please provide a description of the inputs.

File: ea_coreg_fa.m (hunk 1 above)
Comment (ningfei) [reply ea_coreg_fa.m]: Coregister FA image to anatomical anchor image.
Will export FA image from DTI if it doesn't exist yet.

**Comment by laurentheirendt** [comment]
@Daniel-Tojal, may you make the changes here?

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal and @ningfei 👍

**Pull request merged**